### DISCUSSION TITLE
[Core Protocol] <FPS> in case of no defender occur... what will happen next?

### DISCUSSION THREAD

**Discussion by AaronLee22** [ethereum-optimism/developers#541] [Core Protocol] [answered]
### Did you check the documentation?

- [X] Yes

### Did you read the specs?

- [X] Yes
- [ ] No

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

What happens if no defense is presented to a counterclaim against a claim? Will the resolution stage still occur, or will nothing happen? Since no defense is made, will the challenger simply get back the initial bond and waste the transaction fee? I assume the challenger might win by default, but I'm not sure what happens next. Does this imply that a reorganization might occur (which I don't think so...)? There should be on/off-chain verification, but without a defender, I'm unsure about the next steps.

**Comment by hash1go** [comment]
Hello.
If a challenger launches an attack against an output proposal and the defender does not defend, the challenger's claim will be accepted. In this case, the output proposal will be overwritten by the challenger.
Currently, the OP Mainnet is in Stage 1 Rollup, so the final output proposal may be overwritten by the Security Council.

**Reply by AaronLee22** [reply]
Hello,
Thank you for your answer.

As an L2 operator aiming to support FPS, I'm concerned about the low participation in Fault Dispute Games on the network, leading to a lack of voluntary defenders. If defenders do not emerge, the operator would need to directly defend the network.

Defending every game would incur significant costs. Based on my calculations, reaching the maximum depth (73) and defending all claims would require 691 ETH per game.  Additionally, if multiple games are generated, we would need to secure enough ETH to cover all these scenarios, which presents a substantial challenge.

Also, as I asked before this could be highly risky, as the scenario you described earlier could be exploited. Do you have any advice on how to mitigate this risk?

**Reply by hash1go** [reply]
For details on the incentives for honest challengers to participate in the game, please refer to the spec here
https://specs.optimism.io/fault-proof/stage-one/bond-incentives.html

**Reply by AaronLee22** [reply]
I have already gone over this spec document but still it does not answer my questions.... can you help me out?

**Reply by hash1go** [reply]
As I understand it, a claim must be accompanied by a bond, and the gas fees for the dispute game are covered by this bond. Therefore, I believe the issue you are concerned about will not occur.
https://blog.oplabs.co/dispute-game-bond-design/

**Comment by joohhnnn** [comment, marked as answer]
> What happens if no one responds to a counterclaim against a claim?

Firstly, if no one continues to challenge it, it will go into the resolve process as the chess clock runs out.

> Will the challenger simply get back the initial bond and waste the transaction fee?

No, if the challenger succeeds, they not only recover their own bond but also gain the bond posted by the party who submitted the incorrect claim, along with potential future incentives.

> Does this imply that there might be a reorganization?

No, there won’t be any reorganization. When creating the dispute game, an L2 block number is inputted as extra data. By this point, the L2 block is already finalized. If the claim is rejected, it proves the claim was invalid and a new, correct FDG needs to be created.

> There should be on/off-chain verification, but without a defender, I'm unsure about the next steps.

Yes, currently a team operates an op-challenger component responsible for this setup, but of course, anyone can run it and participate in the game.

> We would need to secure enough ETH to cover all these scenarios, which presents a substantial challenge.

Regarding this issue, I think it's more of a logical problem because when you successfully defend or attack, the bond of the losing side is acquired by you. Thus, I don’t believe anyone would intentionally maintain an incorrect claim and deliberately put ETH into this game. If you look at the information on the mainnet, cases that reach the step phase are extremely rare and are very specific cases, so you shouldn’t feel too much pressure.

**Reply by AaronLee22** [reply]
Wow, Thank you for your answer it really helped a lot!
